src/python/test/test_nextflow.py

```diff
@@ -215,6 +215,20 @@ def test_dx_build_nextflow_from_repository_default_metadata(self):
         details = applet.get_details()
         self.assertEqual(details["repository"], hello_repo_url)
 
+    @unittest.skipUnless(testutil.TEST_RUN_JOBS,
+                         'skipping tests that would run jobs')
+    def test_dx_build_nextflow_from_repository_destination(self):
```

Comment:
Thanks for the test. For the case when the project is different from the currently selected ones you could use the temporary_project() we have for a quick creation of a separate project, e.g. like [here](https://github.com/dnanexus/dx-toolkit/blob/980b9b534b977011888860cd680eae4013dc5e6f/src/python/test/test_dxclient.py#L1650). This one is sufficient though, thanks!